images/router/haproxy/conf/haproxy-config.template

```diff
@@ -186,7 +186,14 @@ defaults
 
   {{ if (gt .StatsPort -1) }}
 listen stats
+    {{ if eq "v4v6" $router_ip_v4_v6_mode }}
   bind :{{ if (gt .StatsPort 0) }}{{ .StatsPort }}{{ else }}1936{{ end }}
+  bind :::{{ if (gt .StatsPort 0) }}{{ .StatsPort }}{{ else }}1936{{ end }} v6only
```

Comment:
I see that this implementation is similar to what we have for port `80` and `443` frontends but just out of curisosity: will this be a shorter equivalent of these 2 bind instructions:
```
bind :::{{ if (gt .StatsPort 0) }}{{ .StatsPort }}{{ else }}1936{{ end }} v4v6
```
? Or there are some pitfalls?

Reply:
Using a single `bind` line with `v4v6` is not equivalent.  See https://github.com/openshift/router/pull/83/commits/74c2d9397cb8e92c28642f56e46fd29a08d8f64b.